index.html

```diff
@@ -39,9 +41,15 @@ <h1 class="project-title">{{title}}</h1>
         <br>
       </script>
       <!--END TEMPLATE FOR TESTIMONIALS CONSTRUCTOR AND PAGE-->
+      <!--TEMPLATE FOR AWESOME PTS-->
+      <script id="admin-template" type="text/x-handlebars-template">
+        <li><strong><a href="{{quoteAuthorLink}}">{{name}}</a></strong>:{{awesomePoints}} AWESOME points!</li>
+      </script>
+      <!--END TEMPLATE FOR AWESOME PTS CONSTRUCTOR AND PAGE-->
   </head>
 
   <body>
+    <!--Depending on what page the user is on, have that page's navbar name appear differentl. eg, if they're on Index, make index bold, others not.-->
```

Comment:
That would be a good idea. I wonder how you could accomplish this? Maybe an event listener that, when you click the appropriate nav bar button, changes the text on the index.html's header h1?